### DISCUSSION TITLE
cwl: How to record \Verb and \EscVerb

### DISCUSSION THREAD

**Discussion by muzimuzhi** [texstudio-org/texstudio#1951] [Q&A]
`fvextra` package (hence `minted`) provides verbatim commands with extended syntax:
 - `\Verb` accepts either `\Verb|<code>|` or `\Verb{<code>}` (required `<code>` containing paired curly braces)
 - `\EscVerb` requires `\EscVerb{<code>}`.
 
How to record such commands in cwl file so their verbatim args are highlighted correctly? `\EscVerb{verbatimSymbol}` fails, as [this TeX-SX question](https://tex.stackexchange.com/q/623238) shown. (Maybe it's just not supported.)

**Comment by muzimuzhi** [comment]
Seems it's hardcoded, see line 189 below
https://github.com/texstudio-org/texstudio/blob/f665089b840ee1a080888983610fc601e4d69222/src/latexparser/latexparsing.cpp#L185-L197

**Reply by muzimuzhi** [reply]
Just find `\verb{abc{` is wrongly highlighted as well. Although valid, it's such a wired use case that an editor can never support.

**Comment by sunderme** [comment]
I would suggest to define it as \Verb{def}.
It will not highlight as verbatim, but it will not mess up syntax checking either.
